services/alertmanager/alertmanager.go

```diff
@@ -73,12 +81,50 @@ func (svc *Service) Run(ctx context.Context) {
 	}
 }
 
+// createDataDir creates alert manager directories if not exists in the persistent volume.
+func (svc *Service) createDataDir() {
+	group, err := user.Lookup("pmm")
+	if err != nil {
+		svc.l.Errorf("error looking up pmm user user info %v.", err)
+		return
+	}
+
+	uid, _ := strconv.Atoi(group.Uid)
+	gid, _ := strconv.Atoi(group.Gid)
+
+	stat, err := os.Stat(alertmanagerDataDir)
+	if os.IsNotExist(err) {
```

Comment:
I think we should do something like that
```
if err == nil {
  return
}

// do everything: mkdir, chown, chmod, etc.

Reply:
stat, err := os.Stat(alertmanagerDataDir)
if err == nil {
  return
}
In such case we cannot handle https://github.com/percona/pmm-managed/pull/388#discussion_r425170314

Reply:
I think @AlekSi means that if err != nil it's allways not exist error based on condition above.

Reply:
Do we really need this `if`? MkdirAll does nothing if directory is exists and doesn't return error.
So we can move check from https://github.com/percona/pmm-managed/pull/388/files#diff-a8171d88a4e24ab2acd4b471b52230faR124 to https://github.com/percona/pmm-managed/pull/388/files#diff-a8171d88a4e24ab2acd4b471b52230faR107